packages/contracts-bedrock/test/L2/L2Genesis.t.sol

```diff
@@ -158,6 +163,63 @@ contract L2GenesisTest is Test {
         withTempDump(_test_allocs_size);
     }
 
+    /// @notice Tests that the L1Block predeploy has the correct config values.
+    function test_config_values_works() external {
+        DeployConfig cfg = genesis.cfg();
+        L1Dependencies memory deps = _dummyL1Deps();
+
+        genesis.cfg().setFundDevAccounts(false);
+        genesis.runWithLatestLocal(deps);
```

Comment:
Seems like `runWithLatestLocal` is using the `_populateNetwork` as true so disregard my previous comment